hassio/addon-store/hassio-addon-store.html

```diff
@@ -41,7 +41,7 @@
   ready() {
     super.ready();
     this.addEventListener('hass-api-called', ev => this.apiCalled(ev));
-    this.addEventListener('hassio-store-refresh', ev => this.refreshData(ev));
+    document.addEventListener('hassio-store-refresh', ev => this.refreshData(ev));
```

Comment:
Uuuuhh.

Data should flow from parents to children. And children can notify parents via events that data should be refreshed. The fact that you have to listen to document means that a parent or sibling is firing an event - that's wrong.

Reply:
I could use `this.$.store.refresh()` instead?

Reply:
Yes! 👍 

Parents that want to call things on their children call the methods directly.